src/proto/graplinc/grapl/api/services/v1beta1/types.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package graplinc.grapl.api.services.v1beta1;
+
+// A protobuf, encoded by splitting the 128bit representation into two 64bit values
+message ProtoUuidV4 {
```

Comment:
Maybe just `Uuid`? `Proto` is redundant because it's a protobuf and `V4` is redundant because it's already included in the serialized data.

Reply:
True about V4, mostly the 'Proto' is just to avoid name collisions - like I'm going to end up importing it `as ProtoUuid` everywhere where I use `use uuid::Uuid`. I don't care though, can change if you'd like. Removing V4 seems fine.

Reply:
It should be fine to just refer to it as `rust_proto::Uuid`, right? In any case, I think the type _here_ should just be `Uuid` to avoid redundancy, and consumers can rename it however they like.